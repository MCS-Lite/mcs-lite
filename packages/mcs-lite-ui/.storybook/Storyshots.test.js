import initStoryshots from 'storyshots';

/**
 * Ignore Snapshot testing with "[Skip]" in story name.
 *
 * @author Michael Hsu
 */

initStoryshots({
  storyRegex: /^((?!\[Skip\]).)*$/,
});
