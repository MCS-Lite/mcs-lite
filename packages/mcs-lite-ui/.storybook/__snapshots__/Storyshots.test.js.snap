exports[`Storyshots API [JSON] kind 1`] = `
<div>
  <pre>
    {
      \"data\": [
        \"default\",
        \"primary\",
        \"success\",
        \"error\",
        \"warning\"
      ]
    }
  </pre>
</div>
`;

exports[`Storyshots API [JSON] theme 1`] = `
<div>
  <pre>
    {
      \"data\": {
        \"color\": {
          \"black\": \"#353630\",
          \"grayBase\": \"#999A94\",
          \"grayDark\": \"#D1D2D3\",
          \"grayLight\": \"#FAFAFA\",
          \"white\": \"#FFFFFF\",
          \"default\": \"#FAFAFA\",
          \"primary\": \"#00A1DE\",
          \"success\": \"#69BE28\",
          \"error\": \"#F5364E\",
          \"warning\": \"#F39A1E\"
        },
        \"base\": {
          \"bodyBackground\": \"#F1F2F7\",
          \"fontSize\": \"14px\"
        },
        \"fontSize\": {
          \"small\": \"86%\",
          \"p\": \"1rem\",
          \"h6\": \"1rem\",
          \"h5\": \"1rem\",
          \"h4\": \"1.14rem\",
          \"h3\": \"1.43rem\",
          \"h2\": \"1.71rem\",
          \"h1\": \"2.43rem\"
        }
      }
    }
  </pre>
</div>
`;

exports[`Storyshots Button API 1`] = `
<button
  className="jRTTze"
  kind="primary">
  Simple button
</button>
`;

exports[`Storyshots Button Overriding style 1`] = `
<button
  className="cqoeNx"
  kind="primary"
  onClick={[Function]}>
  Overriding style
</button>
`;

exports[`Storyshots Button With kind props 1`] = `
<div
  className="ekbxOp">
  <button
    className="fQCDqY"
    kind="default"
    onClick={[Function]}>
    default
  </button>
  <button
    className="jRTTze"
    kind="primary"
    onClick={[Function]}>
    primary
  </button>
  <button
    className="dKIIwG"
    kind="success"
    onClick={[Function]}>
    success
  </button>
  <button
    className="cbvlGV"
    kind="error"
    onClick={[Function]}>
    error
  </button>
  <button
    className="caiLV"
    kind="warning"
    onClick={[Function]}>
    warning
  </button>
</div>
`;

exports[`Storyshots Button With theme provider (Material) 1`] = `
<div
  className="ekbxOp">
  <button
    className="fLAIJr"
    kind="default"
    onClick={[Function]}>
    default
  </button>
  <button
    className="jBPAeu"
    kind="primary"
    onClick={[Function]}>
    primary
  </button>
  <button
    className="ddoWjI"
    kind="success"
    onClick={[Function]}>
    success
  </button>
  <button
    className="gGdFcO"
    kind="error"
    onClick={[Function]}>
    error
  </button>
  <button
    className="hYHCGC"
    kind="warning"
    onClick={[Function]}>
    warning
  </button>
</div>
`;

exports[`Storyshots Card API 1`] = `
<div
  className="dVFvPA">
  Card Content
</div>
`;

exports[`Storyshots DataChannelCard API 1`] = `
<div
  className="jFlOlG">
  Content
</div>
`;

exports[`Storyshots Heading API 1`] = `
<h1
  className="fVtTCn"
  color="black">
  Level 1
</h1>
`;

exports[`Storyshots Heading With color props 1`] = `
<div>
  <h2
    className="yHLyb"
    color="black">
    Level 2 
    black
  </h2>
  <h2
    className="dKqSmo"
    color="grayBase">
    Level 2 
    grayBase
  </h2>
  <h2
    className="ciUZCF"
    color="grayDark">
    Level 2 
    grayDark
  </h2>
  <h2
    className="eOEYXe"
    color="grayLight">
    Level 2 
    grayLight
  </h2>
  <h2
    className="gLUCQb"
    color="white">
    Level 2 
    white
  </h2>
  <h2
    className="eOEYXe"
    color="default">
    Level 2 
    default
  </h2>
  <h2
    className="fsfMJE"
    color="primary">
    Level 2 
    primary
  </h2>
  <h2
    className="dsFKDb"
    color="success">
    Level 2 
    success
  </h2>
  <h2
    className="fzlnFI"
    color="error">
    Level 2 
    error
  </h2>
  <h2
    className="dEVvxg"
    color="warning">
    Level 2 
    warning
  </h2>
</div>
`;

exports[`Storyshots Heading With level props 1`] = `
<div>
  <h1
    className="fVtTCn"
    color="black">
    Level 
    1
  </h1>
  <h2
    className="yHLyb"
    color="black">
    Level 
    2
  </h2>
  <h3
    className="jpmjeD"
    color="black">
    Level 
    3
  </h3>
  <h4
    className="egURlk"
    color="black">
    Level 
    4
  </h4>
  <h5
    className="ljcViA"
    color="black">
    Level 
    5
  </h5>
  <h6
    className="ljcViA"
    color="black">
    Level 
    6
  </h6>
</div>
`;

exports[`Storyshots P API 1`] = `
<p
  className="eSOTAI"
  color="black">
  Nisi eu eiusmod cupidatat aute laboris commodo excepteur esse dolore incididunt incididunt aliquip pariatur est minim officia sit. Nulla pariatur duis duis quis commodo cupidatat voluptate enim culpa elit adipisicing do cupidatat sint anim. Cillum elit magna occaecat proident sit cupidatat ad quis sunt id culpa culpa. Ad duis nulla in incididunt amet consequat officia ad voluptate voluptate. Pariatur eiusmod ullamco cupidatat non magna officia aute magna deserunt qui aute dolor eu. Qui amet non ex cillum sunt ad velit consequat ipsum velit.
</p>
`;

exports[`Storyshots P With color props 1`] = `
<div>
  <p
    className="eSOTAI"
    color="black">
    Paragraph 
    black
  </p>
  <p
    className="WEOUa"
    color="grayBase">
    Paragraph 
    grayBase
  </p>
  <p
    className="jAIFMf"
    color="grayDark">
    Paragraph 
    grayDark
  </p>
  <p
    className="kXKPlg"
    color="grayLight">
    Paragraph 
    grayLight
  </p>
  <p
    className="grYLXw"
    color="white">
    Paragraph 
    white
  </p>
  <p
    className="kXKPlg"
    color="default">
    Paragraph 
    default
  </p>
  <p
    className="dlDdTl"
    color="primary">
    Paragraph 
    primary
  </p>
  <p
    className="dkpTNT"
    color="success">
    Paragraph 
    success
  </p>
  <p
    className="cXfdto"
    color="error">
    Paragraph 
    error
  </p>
  <p
    className="ipWZDW"
    color="warning">
    Paragraph 
    warning
  </p>
</div>
`;

exports[`Storyshots Switch API 1`] = `
<div>
  <div
    checked={false}
    className="cICTqL" />
  <div
    checked={true}
    className="jvOKcu" />
</div>
`;

exports[`Storyshots Switch Overriding style 1`] = `
<div
  checked={false}
  className="hZJZFY"
  onClick={[Function]} />
`;

exports[`Storyshots Switch Stateful switch 1`] = `
<div
  checked={false}
  className="cICTqL"
  onClick={[Function]} />
`;
